lib/engines/content_block_manager/app/models/content_block_manager/content_block/document/scopes/searchable_by_updated_date.rb

```diff
@@ -0,0 +1,10 @@
+module ContentBlockManager
+  module ContentBlock::Document::Scopes::SearchableByUpdatedDate
+    extend ActiveSupport::Concern
+
+    included do
+      scope :from_date, ->(date) { where("content_block_documents.updated_at >= ?", date) }
```

Comment:
so it looks like editing an Edition updates the Document (even if we never get to the publish/`latest_edition` stage).
I think a solution would be doing a query here to get the current `latest_edition` (which should always be the latest published/scheduled on) and checking that date.